### ISSUE TITLE
BUG - [AMANDA] Folder In Date Rule w/ Internet Pending Status (On PROD🎉🎉)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#8693]
> What application are you using?

AMANDA

> Describe the problem.
Peter M.
Hi all, 2022 024861 000 00 EX was in Application Incomplete since 02/22/22. Yesterday it went into Internet Pending and now it's showing an "In Date" of 02/22/22. 

Last year, we corrected the system so that the "In Date" reflected the date that the Applicant *actually* submitted the request (as opposed to the date that they began the application).

At some point within the past couple of months this fix was lost.  This causes data infidelity and invalidates our permitting measurements.

Internet Browser: Internet Explorer

> Requested By
Paloma A.

Request ID: DTS22-103625

**Comment by TracyLinder** [comment]
@Nadin-Nader this looks to be a bug related to #1726, which was promoted to PROD in May 2021. I'll put this on deck.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I checked this on TEST, it changed the indate with Internet Pending(Was in Pending Payment status and with payment, changed to Internet Pending and also changed the indate as expected).
RSN - 12793518

Is there any other scenarios that EX turns into INTERNET PENDING, other than payment, lets say any process attempt? 
Any idea, in PROD example 2022 024861 000 00 EX, what triggered the status change to INTERNET PENDING?

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0207022 Is submitted for DEV deployment.

**Comment by Nadin-Nader** [comment]
@SubhaShroff this has been deployed to DEV please UAT

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0207689 is submitted for TEST deployment.

**Comment by SubhaShroff** [comment]
### Status:
Waiting on CTM to do TEST promotion.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I tested this ticket on TEST for EX. It works as expected. Ready for UAT.

**Comment by Nadin-Nader** [comment]
##  Approved for PROD🎉🎉

**Comment by SubhaShroff** [comment]
@Nadin-Nader SCTASK0208504 is submitted for PROD deployment.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  this ticket is deployed in PROD.

**Comment by SubhaShroff** [comment]
### Status:
This was deployed in PROD 2 weeks ago. Waiting to close.

**Comment by TracyLinder** [comment]
Paloma is asking staff if they've seen this issue happening recently.

**Comment by Nadin-Nader** [comment]
> [Yesterday 7:02 PM] Amayo-Ryan, Paloma
>     Nader, Nadin for 8108 BUG, I believe Linder, Tracy tested that during our Friday meeting and confirmed that it was all systems go.  I want to say that she tested another card as well but I can't remember now which one.
> 
> For 7912 everything appears to be working fine so I'm a-okay with closing it out.
> 
> For 8693 it's not something we can exactly test but we haven't seen any issues in a couple of weeks so hopefully that's a sign that the In Date issues are fixed (again)!

Closing card
